wpeverest/everest-forms
# Tweak - Design enhancement

### All Submissions:






### Changes proposed in this Pull Request:



Closes #82 , #114  .

### How to test the changes in this Pull Request:

1. Check the smart-tag design in Windows and iOS safari browser.
2. Check the font size differences between field label and field description in form builder.

### Types of changes:




### Other information:




### Changelog entry

>Tweak - Smart-tag design enhancement
